nixd/lib/Eval/AttrSetProvider.cpp

```diff
@@ -131,24 +132,25 @@ void fillOptionDescription(nix::EvalState &State, nix::Value &V,
   fillOptionDeclarations(State, V, R);
   // FIXME: add definitions location.
   if (V.type() == nix::ValueType::nAttrs) [[likely]] {
-    assert(V.attrs);
-    if (auto *It = V.attrs->find(State.symbols.create("type"));
-        It != V.attrs->end()) [[likely]] {
+    assert(V.attrs());
+    if (auto *It = V.attrs()->find(State.symbols.create("type"));
+        It != V.attrs()->end()) [[likely]] {
       OptionType Type;
       fillOptionType(State, *It->value, Type);
       R.Type = std::move(Type);
     }
 
-    if (auto *It = V.attrs->find(State.symbols.create("example"));
-        It != V.attrs->end()) {
+    if (auto *It = V.attrs()->find(State.symbols.create("example"));
+        It != V.attrs()->end()) {
       State.forceValue(*It->value, It->pos);
 
       // In nixpkgs some examples are nested in "literalExpression"
       if (nixt::checkField(State, *It->value, "_type", "literalExpression")) {
         R.Example = nixt::getFieldString(State, *It->value, "text");
       } else {
         std::ostringstream OS;
-        It->value->print(State.symbols, OS);
+        // TODO: replace this with something that can traverse the symbol table
+        //It->value->print(State.symbols, OS);
```

Comment:
@inclyc unsure what to do here. We need symbols to be looked up here in the table. I suppose they should come from `IT` somehow?

Reply:
maybe just:

```C++
It->value->print(State, OS);
```